### DISCUSSION TITLE
Telescope Fzf-Native

### DISCUSSION THREAD

**Discussion by msMatix** [LazyVim/LazyVim#3156] [General]
Hello everyone :),

I have seen in the docs that LazyVim as Recipe has:
```
{
  "telescope.nvim",
  dependencies = {
    "nvim-telescope/telescope-fzf-native.nvim",
    build = "make",
    config = function()
      require("telescope").load_extension("fzf")
    end,
  },
}
```

However, if I look in the file `../plugins/editor.lua`, I can see that fzf native is already set up here. Is the recipe still necessary afterward?

Many thanks in advance!
Matix

**Comment by dpetka2001** [comment]
I don't believe that's necessary.

**Reply by msMatix** [reply]
Alright. I think the same. I also looked into the loaded plugins and without the recipe the telescope fzf plugin is loaded. 
Thanks!
